tauri-app/src/lib/components/ModalExecute.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
```

Comment:
I think ideally this component would behave more like the other generic InputX components, where they expose a single prop 'value' and provide a generic input to choose a wallet of any type. It would require some more refactoring of how we're handling wallets in the store though. I'll make a separate issue for it. For now I think this is good.

Reply:
#481